docs/extend/admin.md

```diff
@@ -124,13 +136,18 @@ app.initializers.add('interstellar', function(app) {
         icon: 'fas fa-rocket', // Font-Awesome Icon
         label: app.translator.trans('acme-interstellar.admin.permissions.fly_rockets_label'), // Permission Label
         permission: 'discussion.rocket_fly', // Actual permission name stored in database (and used when checking permission).
+        tagScoped: false, // Should it be possible to apply this permission on tags, not just globally?
```

Comment:
I'm confused by the way this is phrased as a question, it looks like this was copied from unfinished code with a TODO.

Maybe "Disallow applying this permission individually on each tag"

It might still be confusing since the way it works is explained below the example...

I think it might be more logical to show an example with `tag:Scoped: true` and just explain why you might want to use `false` in the documentation below (as is already the case)